lively.morphic/svg-morph/svg-morph.js

```diff
@@ -96,17 +92,15 @@ export class SVGMorph extends Morph {
       this.createSVGSelectionBox();
     } else {
       this.removeSVGSelection();
-      if (this.target && this.target.selected) {
-        this.removePathSelection();
-      }
+      this.removePathSelection();
       this.removeAllControlPoints();
-      this.target.selected = false;
```

Comment:
I am not allowed to make a suggestion for that but I think that we can just call `this.removeAllSelections` if here editMode is false anyways and we are removing all Selections